Float64 vel1
Float64 vel2
Float64 vel3
Float64 vel4


